Win32/config/_config.hpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2010-2022, Lawrence Livermore National Security, LLC. Produced
```

Comment:
Is there no way to somehow reuse `config/config.hpp.in` here? Maybe generate it as we do in `make config`? The version and git hashes in particular are not accurate and we will have to remember to update them.